clients/src/main/java/org/apache/kafka/common/network/Selector.java

```diff
@@ -367,19 +413,25 @@ private void pollSelectionKeys(Iterable<SelectionKey> selectionKeys,
                                 socketChannel.socket().getSendBufferSize(),
                                 socketChannel.socket().getSoTimeout(),
                                 channel.id());
+                        madeReadProgressLastPoll = true;
```

Comment:
Is there a need to set madeReadProgressLastPoll here? It seems we only need to set when doing reads?